docs/api/select.md

```diff
@@ -83,6 +83,34 @@ import UsingCompareWithExample from '@site/static/usage/v7/select/objects-as-val
 import ObjectValuesAndMultipleSelectionExample from '@site/static/usage/v7/select/objects-as-values/multiple-selection/index.md';
 
 <ObjectValuesAndMultipleSelectionExample />
+  
+
+## Label Positioning
+
+Labels will take up the width of their content by default. This positioning can be changed to be a fixed width, stacked, or floating label.
+
+import LabelExample from '@site/static/usage/v7/select/labels/index.md';
+
+<LabelExample />
```

Comment:
Stacked and floating labels look/behave the same if the selects have placeholders; maybe those two should skip them so the difference is more clear?

Reply:
Good call. I fixed this in https://github.com/ionic-team/ionic-docs/pull/2719/commits/fd8b0b13611785aeee334edad8c0638317770456